tests/integration/test_labeling_frontend.py

```diff
@@ -12,6 +12,8 @@ def test_get_labeling_frontends(client):
         where=LabelingFrontend.iframe_url_path ==
         target_frontend.iframe_url_path)
     for frontend in filtered_frontends:
+        if frontend.name != 'Editor':
```

Comment:
do we know that frontends[0] will always be a non V4 frontend?

Reply:
oops. this wasn't supposed to be pushed. Yeah this is broken.